ambry-cloud/src/test/java/com.github.ambry.cloud/azure/AzureIntegrationTest.java

```diff
@@ -249,6 +250,56 @@ public void testPurgeDeadBlobs() throws Exception {
     cleanup();
   }
 
+  /**
```

Comment:
Test coverage %?

Reply:
Not following.  Are you asking what the test coverage is or saying it should be mentioned in comments?

Reply:
Sorry, just asking what it is for the intended tested object, or how much of the newly added code is tested, can be put in the comments.

Reply:
For both AzureCloudDestination and CloudBlobStore, the new method has 85-90% coverage, only  exception throw is missing.